### PR TITLE
Updated to Player Core changes

### PR DESCRIPTION
**Pull request by balavor**
@VerizonAdPlatforms/video-partner-sdk-developers: Please review.

### CODE REVIEW

File: sources/advertisements/MidrollDetectorTests.swift [hunk 1]
```diff
@@ -36,9 +36,11 @@ class MidrollDetectorTests: QuickSpec {
                         case (let targetAdRequest as AdRequest, let recordAdRequest as AdRequest):
                             return targetAdRequest.id == recordAdRequest.id
                         case (let targetShowAd as ShowAd, let recordShowAd as ShowAd):
-                            guard case .mp4(let targetShowCreative) = targetShowAd.creative else { return false }
-                            guard case .mp4(let recordShowAdCreative) = recordShowAd.creative else { return false }
-                            return targetShowCreative.url == recordShowAdCreative.url
+                            guard case .mp4(let targetShowAdCreatives) = targetShowAd.creative else { return false }
```
Comment (AndriiMoskvin) [inline sources/advertisements/MidrollDetectorTests.swift]: what is `targetShowAd` and `recordShowAd`?

Comment (VladyslavAnokhin) [inline sources/advertisements/MidrollDetectorTests.swift]: You can use one guard in this case

**Review by AndriiMoskvin — DISMISSED** [review]

File: sources/advertisements/MidrollDetectorTests.swift (hunk 1 above)
Comment (balavor) [reply sources/advertisements/MidrollDetectorTests.swift]: Done

Comment (balavor) [reply sources/advertisements/MidrollDetectorTests.swift]: These are expected and resulted actions

**Review by VladyslavAnokhin — DISMISSED** [review]

**Review by AndriiMoskvin — DISMISSED** [review]

**Review by VladyslavAnokhin — DISMISSED** [review]

**Review by VladyslavAnokhin — APPROVED** [review]

**Pull request merged**